services/server/server.go

```diff
@@ -418,8 +418,10 @@ func LoadPlugins(ctx context.Context, config *srvconfig.Config) ([]*plugin.Regis
 
 			path := filepath.Join(ic.Root, "meta.db")
 			ic.Meta.Exports["path"] = path
-
-			db, err := bolt.Open(path, 0644, nil)
+			options := &bolt.Options{
+				Timeout: timeout.DefaultTimeout,
```

Comment:
`DefaultTimeout` is currently 1 second. Is it enough?

Reply:
if the blot file is not locked, `blot.Open` will only takes a few milliseconds. I think 1 sencond is enough for that.

Reply:
I'm a bit hesitant to hardcode 1 second here. I'd suggest to use our `timeout` package, so  clients can configure this parameter if needed. We can leave 1 second as default parameter though. 

example:
https://github.com/containerd/containerd/blob/main/runtime/v2/shim.go#L47
https://github.com/containerd/containerd/blob/main/runtime/v2/shim.go#L53

Reply:
Done

Reply:
> so clients can configure this parameter if needed

Does this means clients to use `go build -ldflags=` to set the value of the consts?